e2e-tests/manifests/rapidast-vapi-pod.yaml

```diff
@@ -29,6 +29,12 @@ spec:
     volumeMounts:
     - name: results
       mountPath: /opt/rapidast/results
+  - command: ["bash", "-c", "cat /opt/rapidast/results/*/*/zap/zap-report.sarif.json"]
```

Comment:
Afaik, we should validate the SARIF report generated by RapiDAST, not the one produced directly by ZAP. The  file is `rapidast-scan-results.sarif`, which merges all SARIF reports from each scanner into a single merged report.